spring-petclinic-config-server/pom.xml

```diff
@@ -9,14 +9,15 @@
 	<description>Spring PetClinic Config Server</description>
 
     <parent>
-        <groupId>org.springframework.samples</groupId>
-        <artifactId>spring-petclinic-microservices</artifactId>
-        <version>2.3.1</version>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.3.1.RELEASE</version>
```

Comment:
No shared parent anymore since each microservice has a completely separate lifecycle now. Some microservices may want to stay on 2.3.1, while others may want to upgrade to 2.3.2 and JDK 14. It becomes a problem when they are bound to the same upstream.

Reply:
Ok, let's go with this choice.